### ISSUE TITLE
Error open dialog

### ISSUE THREAD

**Issue by tonal** [eyalroz/removedupes#121]
[v0.5.2](https://github.com/eyalroz/removedupes/releases/tag/v0.5.2)
Thunderbird 102.9.1 (64-bit)
```
Ошибка синтаксического анализа XML: неопределённая сущность
Адрес: chrome://removedupes/content/removedupes-dialog.xhtml
Строка 150, символ 13:
            <radio id="move_to_common_account_trash_action"
------------^
```

**Comment by eyalroz** [comment]
You are absolutely right, the release version doesn't have the localization strings for this new action! Please try this build.
[removedupes_0.5.2.1b1_tbird.zip](https://github.com/eyalroz/removedupes/files/11198024/removedupes_0.5.2.1b1_tbird.zip)

**Comment by eyalroz** [comment]
Also... would you be interested in helping me actually finalizing the translation into Russian? I used to use this translation website called babelzilla.org , but it's been sort of dead for the past few years, so now I have to use English as "fake Russian" for new strings.

**Comment by eyalroz** [comment]
Anyway, fixed this for 0.5.2.1, albeit with English stand-ins for most languages.
